tfjs-backend-webgpu/src/kernels/Abs.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import {KernelConfig, KernelFunc, Abs} from '@tensorflow/tfjs-core';
+import {unaryKernelFunc} from '../kernel_utils/kernel_funcs_utils';
+import {ABS} from './unary_op_webgpu';
+import {simpleAbsImplCPU} from '../kernel_utils/shared';
+
+export const abs =
+    unaryKernelFunc({opSnippet: ABS, cpuKernelImpl:simpleAbsImplCPU});
+
+export const absConfig: KernelConfig = {
+  kernelName: Abs,
+  backendName: 'webgpu',
+  kernelFunc: abs as {} as KernelFunc
```

Comment:
nit: These casts seem like they shouldn't be necessary? But if they are, no problem.